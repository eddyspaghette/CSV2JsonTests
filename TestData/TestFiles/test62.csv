ッ;,
ツ;ぃ,ぎ,せ,℟,▓
ヅ,い,け,₠,Å,○
ミ,ぅ,,₡,ℬ,☝
デ,う,こ,,Δ,⭐
ト,ぇ,ご,₣,ψ,😵
ド,え,さ,℀,ↂ,
