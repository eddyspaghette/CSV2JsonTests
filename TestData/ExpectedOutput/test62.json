[
 {
   "ッ;": "ツ;ぃ",
   "FIELD2": "ぎ",
   "FIELD3": "せ",
   "FIELD4": "℟",
   "FIELD5": "▓",
   "FIELD6": ""
 },
 {
   "ッ;": "ヅ",
   "FIELD2": "い",
   "FIELD3": "け",
   "FIELD4": "₠",
   "FIELD5": "Å",
   "FIELD6": "○"
 },
 {
   "ッ;": "ミ",
   "FIELD2": "ぅ",
   "FIELD3": "",
   "FIELD4": "₡",
   "FIELD5": "ℬ",
   "FIELD6": "☝"
 },
 {
   "ッ;": "デ",
   "FIELD2": "う",
   "FIELD3": "こ",
   "FIELD4": "",
   "FIELD5": "Δ",
   "FIELD6": "⭐"
 },
 {
   "ッ;": "ト",
   "FIELD2": "ぇ",
   "FIELD3": "ご",
   "FIELD4": "₣",
   "FIELD5": "ψ",
   "FIELD6": "😵"
 },
 {
   "ッ;": "ド",
   "FIELD2": "え",
   "FIELD3": "さ",
   "FIELD4": "℀",
   "FIELD5": "ↂ",
   "FIELD6": ""
 }
]